addons/website/static/src/js/post_link.js

```diff
@@ -9,6 +9,22 @@ publicWidget.registry.postLink = publicWidget.Widget.extend({
     events: {
         'click': '_onClickPost',
     },
+
+    /**
+     * Allows the link to be interacted with only when Javascript is loaded.
+     *
+     * @override
+     */
+    start() {
+        const def = this._super(...arguments);
+        this.el.classList.add('o_post_link_js_loaded');
+        return def;
```

Comment:
```suggestion
        this.el.classList.add('o_post_link_js_loaded');
        return this._super(...arguments);
```
But not strictly needed, as you prefer. If you keep your form, rename the `def` into `prom`. `def` was what we used when using jQuery's Deferred objects. Now we use native Promises.

Reply:
No specific reason to wait for the super so I'll go with the shorted version :+1: